src/routes/onboarding/glucose-notification-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { Form } from "../../components";
+
+interface Props {
+  index?: number;
+  onDelete?: (index: number) => void;
+  elementId?: number;
+}
+
+export const GlucoseNotificationList: React.FC<Props> = ({ index }) => {
+  return (
+    <div className="flex flex-col max-w-screen-md bg-lightPink overflow-y-auto relative m-auto h-screen">
+      <div className="flex flex-col self-center gap-6 items-center px-4 max-w-[500px]">
+        <h2 className="font-text text-section-header font-medium leading-[38.4px] text-center max-w-[358px] mx-auto">
```

Comment:
I might be off here, but the color needs to be #121212. 
Add text-black to achieve that.